tests/utils.go

```diff
@@ -1760,6 +1760,31 @@ func LoggedInFedoraExpecter(vmi *v1.VirtualMachineInstance) (expect.Expecter, er
 	return expecter, err
 }
 
+func LoggedInFedoraExpecter(vmi *v1.VirtualMachineInstance) (expect.Expecter, error) {
```

Comment:
I think that such a function got merged a few days  ago in another PR: #1543. Can we merge them?

Reply:
Nice. Adopted.